{"uid":"2155d04f9b595a97","name":"A Service Provider tries to activates a debtor without having the correct credentials","fullName":"Service Providers activate debtors: A Service Provider tries to activates a debtor without having the correct credentials","historyId":"6197200913c3f4cb99beaebefbac025c","time":{"start":1736320038247,"stop":1736320038656,"duration":409},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/activation_steps.py\", line 26, in then_sp_fails_activation_wrong_credentials\n    assert context.activation_response == 403\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/activation_steps.py\", line 26, in then_sp_fails_activation_wrong_credentials\n    assert context.activation_response == 403\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Given the debtor A has fiscal code random","time":{"start":1736320038247,"stop":1736320038247,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the Service Provider tries to activate the debtor A","time":{"start":1736320038247,"stop":1736320038655,"duration":408},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the activation fails because the Service Provider has wrong credentials","time":{"start":1736320038655,"stop":1736320038656,"duration":1},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/activation_steps.py\", line 26, in then_sp_fails_activation_wrong_credentials\n    assert context.activation_response == 403\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"activation"},{"name":"tag","value":"authentication"},{"name":"tag","value":"unhappy_path"},{"name":"feature","value":"Service Providers activate debtors"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["activation","unhappy_path","authentication"]},"source":"2155d04f9b595a97.json","parameterValues":[]}